spark/src/main/scala/vegas/spark/Spark.scala

```diff
@@ -4,14 +4,11 @@ import org.apache.spark.rdd.RDD
 import org.apache.spark.sql.DataFrame
 
 import vegas.DSL._
-
-/**
-  * @author Aish Fenton.
-  */
+import vegas.DSL.DataDSL
 
 object Spark {
 
-  implicit class SparkExt(val specBuilder: ExtendedUnitSpecBuilder) {
+  implicit class SparkExt(val specBuilder: DataDSL[_]) {
```

Comment:
I think it will work. See https://github.com/apache/spark/blob/master/mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala#L317 for example.